.github/scripts/print_latest_commits.py

```diff
@@ -5,14 +5,14 @@
 import os
 import re
 
-regex = [
-    "^pull+",
-    "^trunk+",
-    "^lint+",
-    "^linux-binary+",
-    "^android-tests+",
-    "^windows-binary+"
-]
+regex = {
```

Comment:
The move may be to move this back to a local variable under isGreen as this dictionary changes now.